db/migrate/20160907095103_add_availability_to_listing_shape.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToListingShape < ActiveRecord::Migration
+  def change
+    add_column :listing_shapes, :availability, :string, limit: 32, default: "none", after: :shipping_enabled
```

Comment:
A MySQL ENUM? We haven't been using that anywhere. Quick google revealed it's a bit controversial topic. The main worries are perf and hard to add new types. I'd feel it's not worth it at least right now but would take more studying if it's a column type we want to use or not.